cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java

```diff
@@ -307,6 +310,18 @@ public boolean accept(String className, URL classUrl, URL classPathUrl) {
     }
     List<String> orderedDependenciesStr = builder.build();
 
+    // These dependency files need to be copied over to spark container
+    System.setProperty(BaseHiveExploreService.SPARK_YARN_DIST_FILES,
```

Comment:
I suppose these system properties are only set on the Explore YARN container, right? As long as they are not set in the Master JVM, it is ok.